### DISCUSSION TITLE
自动伐木无法正常运行

### DISCUSSION THREAD

**Discussion by q-zx-9** [babalae/better-genshin-impact#849] [General]
无法检测到王树瑞佑已装备

**Comment by q-zx-9** [comment]
<img width="937" alt="Desktop Screenshot 2024 12 11 - 09 35 46 12" src="https://github.com/user-attachments/assets/f884c36f-0421-4739-a910-fb7dbb3de069">
补充图片

**Comment by huiyadanli** [comment]
请阅读 https://bgi.huiyadan.com/quickstart.html 进行排查

出现这种情况原因大概率是以下情况中的一个或者多个

1. 截图器无法正常获取图像
2. 游戏画面分辨率不是16:9
3. 游戏画面带有滤镜或者开启了HDR等
